build.gradle.kts

```diff
@@ -24,10 +29,19 @@ plugins {
     id("org.jlleitschuh.gradle.ktlint") version "10.1.0"
 }
 
```

Comment:
Husk å oppdatere README med det som kreves for å kunne hente pakker fra Github Package Registry